### PR TITLE
add note about 3rd party AV compatibility

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8695#issuecomment-738723316

### CODE REVIEW

**Comment by nam31** [comment]
@JohanFreelancer9 can you please do the copy editor review for this PR
Thank you

File: windows/security/threat-protection/microsoft-defender-antivirus/microsoft-defender-antivirus-compatibility.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ If you uninstall the other product, and choose to use Microsoft Defender Antivir
...
 
 > [!IMPORTANT]
-> If you are using [Microsoft Endpoint DLP](https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-atp/information-protection-in-windows-overview), Microsoft Defender Antivirus real-time protection is enabled, even when Microsoft Defender Antivirus is running in passive mode. Endpoint DLP depends on real-time protection to operate.
+> If you are using [Microsoft Endpoint DLP](https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-atp/information-protection-in-windows-overview), Microsoft Defender Antivirus real-time protection is enabled, even when Microsoft Defender Antivirus is running in passive mode. It won't conflict with 3rd party AV solutions installed on the endpoint. Endpoint DLP depends on real-time protection to operate.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-antivirus/microsoft-defender-antivirus-compatibility.md:98]:
```suggestion
> If you are using [Microsoft Endpoint DLP](https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-atp/information-protection-in-windows-overview), Microsoft Defender Antivirus real-time protection is enabled, even when Microsoft Defender Antivirus is running in passive mode. It won't conflict with third-party antivirus solutions installed on the endpoint. Endpoint DLP depends on real-time protection to operate.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by denisebmsft** [comment]
Thank you @VLG17 and @JohanFreelancer9

**Pull request merged**